sklearn/linear_model/_base.py

```diff
@@ -472,10 +477,11 @@ class LinearRegression(MultiOutputMixin, RegressorMixin, LinearModel):
     """
     @_deprecate_positional_args
     def __init__(self, *, fit_intercept=True, normalize=False, copy_X=True,
-                 n_jobs=None):
+                 positive=False, n_jobs=None):
```

Comment:
Positional arguments are deprecated, but still supported. Therefore, it is still valid to use `LinearRegression(True, False, True, -1)`. I think we should add the new parameter at the end, not to break this behavior. We will be able to reorder the parameters once the positional arguments stop being supported.